static/app/views/performance/cache/tables/transactionsTable.tsx

```diff
@@ -34,7 +34,12 @@ type Row = Pick<
 >;
 
 type Column = GridColumnHeader<
-  'transaction' | 'spm()' | 'cache_miss_rate()' | 'time_spent_percentage()'
+  | 'transaction'
+  | 'spm()'
+  | 'cache_miss_rate()'
+  | 'time_spent_percentage()'
+  | 'project.id'
```

Comment:
```suggestion
  | 'project.id'
```

Not needed anymore, right?

Reply:
👍